services/src/datasets/storage.rs

```diff
@@ -77,6 +81,7 @@ impl From<ImportDataset> for Dataset {
             description: value.description,
             result_descriptor: value.result_descriptor,
             source_operator: value.source_operator,
+            symbology: None,
```

Comment:
We should have at least one test for the dataset and one for the handler I guess.

Reply:
what do you want me to test?

Reply:
We should test the case where we have specified a symbology and return it to the client.
Then, we know when something breaks in our return type.

Reply:
do you mean checking the serialization of a Dataset? I can do it, but we already have tests for Symbology serialization